src/js/simpleSmtpServer.js

```diff
@@ -0,0 +1,86 @@
+// A module to wire in simplesmtp to handle incoming messages.
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.test.mail.smtp.simpleSmtpServer");
+
+var simplesmtp = require("simplesmtp");
+var fs         = require("fs");
+
+gpii.test.mail.smtp.simpleSmtpServer.handleStartData = function (that, connection) {
+    var timestamp = (new Date()).getTime();
+    that.messageFile = that.options.config.outputDir + "/message-" + timestamp + ".txt";
```

Comment:
Should rename this something like "currentMessageFile" to emphasize that it is some volatile piece of state that will keep getting overwritten with each new connection.